src/main/java/athena/tasks/Event.java

```diff
@@ -0,0 +1,36 @@
+package athena.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+// STRETCH GOAL: Add a duration for event?
+public class Event extends Task {
+    private static final DateTimeFormatter outputFormatter =
```

Comment:
perhaps using uppercase and underscore to separate the words for outputFormatter e.g. OUTPUT_FORMATTER helps in conforming to the coding standard